ProjectManager/ProjectManager/Controller/ListViewController.swift

```diff
@@ -0,0 +1,283 @@
+        MockDataManager.shared.todoList
+    }
+    private var doingModels: [TodoModel] {
+        MockDataManager.shared.doingList
+    }
+    private var doneModels: [TodoModel] {
+        MockDataManager.shared.doneList
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationBar()
+        configureContentView()
+    }
+    
+    private func configureNavigationBar() {
+        navigationItem.title = ListViewTitle.navigationBar
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add,
+                                                            target: self,
+                                                            action: #selector(tappedAddButton))
+    }
+    
+    @objc private func tappedAddButton() {
+        let newTodoItem = TodoModel()
+        let todoViewController = TodoViewController()
+        
+        todoViewController.configureAddView(with: newTodoItem)
+        showTodoItemView(nextViewController: todoViewController)
+    }
+    
+    private func showTodoItemView(nextViewController: UIViewController) {
+        let viewController = UINavigationController(rootViewController: nextViewController)
+        
+        viewController.modalPresentationStyle = .formSheet
+        viewController.preferredContentSize = CGSize(width: 650, height: 650)
+        
+        present(viewController, animated: true)
+    }
+    
+    private func configureContentView() {
+        view.backgroundColor = .white
+        setTableViews()
+        configureLayout()
+        applyAllSnapshot()
+        addTodoObserver()
+    }
+    
+    private func setTableViews() {
+        let tableviews = [todoTableView, doingTableView, doneTableView]
+        
+        tableviews.forEach {
+            $0.delegate = self
+            listStackView.addArrangedSubview($0)
+            $0.addGestureRecognizer(UILongPressGestureRecognizer(target: self,
+                                                                 action: #selector(self.pressTableViewCell)))
+        }
+    }
+    
+    private func configureLayout() {
+        view.addSubview(listStackView)
+        
+        NSLayoutConstraint.activate([
+            listStackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            listStackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            listStackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            listStackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)
+        ])
+    }
+    
+    private func addTodoObserver() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(updateView),
+                                               name: Notification.Name.mockModels,
+                                               object: nil)
+    }
```

Comment:
```suggestion
    private func addTodoObserver() {
        NotificationCenter.default.addObserver(
            self,
            selector: #selector(updateView),
            name: Notification.Name.mockModels,
            object: nil
        )
    }
```

요렇게 줄바꿈 하는 경우가 더 많은거같긴해요ㅎㅎ 
이런건 개인 취향이긴하니까 코멘트 남겨만 두겠습니당.